From 7a7f541a952d5ca040cf963e77743ec3164357c3 Mon Sep 17 00:00:00 2001
From: Marc Abramowitz <marc@marc-abramowitz.com>
Date: Thu, 3 Mar 2016 14:40:38 -0800
Subject: [PATCH] No setup_requires of distribute

distribute was long ago merged into setuptools and distribute is no
longer maintained.
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3412c93..5d12635 100644
--- a/setup.py
+++ b/setup.py
@@ -19,6 +19,5 @@
       test_suite='nose.collector',
       packages=['dotenv'],
       tests_require=['nose'],
-      setup_requires=['distribute'],
       scripts=['scripts/dotenv']
      )
